src/main/java/com/hotline/pageobject/modules/HeaderModule.java

```diff
@@ -69,19 +65,23 @@ public HomePage gotoHomePage() {
         return new HomePage(driver);
     }
 
-    public WebElement getChooseLanguage() {
+    public WebElement getChooseLanguageUkr() {
+        chooseLanguage = driver.findElement(By.xpath("//*[@data-language='uk'][1]"));
```

Comment:
You can replace this xpath with CSS selector, for example this one: `.header [data-language="uk"]` 
Just find some header class or element to recognise the first element directly